app/src/main/java/com/bootcamp_android/parking_app/fragments/List.kt

```diff
@@ -17,7 +17,7 @@ private const val ARG_PARAM2 = "param2"
  * Use the [Lista.newInstance] factory method to
  * create an instance of this fragment.
  */
-class Lista : Fragment() {
+class List : Fragment() {
```

Comment:
Android developers write the key word "Fragment" or "Activity" at the end of the class name, for instance:

- ListFragment
- HomeActivity

Reply:
ok